packages/patternfly-4/react-core/src/components/DataList/DataList.md

```diff
@@ -184,6 +225,53 @@ class CheckboxActionDataList extends React.Component {
             </DataListAction>
           </DataListItemRow>
         </DataListItem>
+        <DataListItem aria-labelledby="check-action-item3">
+          <DataListItemRow>
+            <DataListCheck aria-labelledby="check-action-item3" name="check-action-check3" />
+            <DataListItemCells
+              dataListCells={[
+                <DataListCell key="primary content">
+                  <span id="check-action-item3">Primary content - Lorem ipsum</span> dolor sit amet, consectetur
+                  adipisicing elit, sed do eiusmod.
+                </DataListCell>,
+                <DataListCell key="secondary content">
+                  Secondary content. Dolor sit amet, consectetur adipisicing elit, sed do eiusmod.
+                </DataListCell>
+              ]}
+            />
+            <DataListAction
+              className={css(DataListActionVisibility.hiddenOnXl)}
+              aria-labelledby="check-action-item3 check-action-action3"
+              id="check-action-action3"
+              aria-label="Actions"
+            >
+              <Dropdown
+                isPlain
+                position={DropdownPosition.right}
+                isOpen={this.state.isOpen3}
+                onSelect={this.onSelect3}
+                toggle={<KebabToggle onToggle={this.onToggle3} />}
+                dropdownItems={[
+                  <DropdownItem key="pri-action3" component="button">Primary</DropdownItem>,
+                  <DropdownItem key="sec1-action3" component="button">Secondary</DropdownItem>,
+                  <DropdownItem key="sec2-action3" component="button">Secondary</DropdownItem>,
+                  <DropdownItem key="sec3-action3" component="button">Secondary</DropdownItem>,
+                ]}
+              />
+            </DataListAction>
+            <DataListAction
+              className={css(DataListActionVisibility.visibleOnXl, DataListActionVisibility.hidden)}
```

Comment:
The buttons are still visible when i reduce the screen width

Reply:
Thanks. I will be looking into it. :smile:

Reply:
Yep, I found the problem!

Reply:
I am just adding unit tests.